### ISSUE TITLE
Webview find widget doesn't work in OSS builds

### ISSUE THREAD

**Issue by mjbvz** [microsoft/vscode#136476]
**Repro**
1. Using an OSS build
2. Open the markdown preview and try triggering find

**Bug**
Nothing happens

Find works fine in insiders/stable

**Comment by mjbvz** [comment]
Sounds like we need to backport the electron changes that enabled this back to the public electron repro

**Comment by deepak1556** [comment]
Upstream PR never got forward https://github.com/electron/electron/pull/28274 pending discussion with chromium https://bugs.chromium.org/p/chromium/issues/detail?id=1189789

**Comment by rzhao271** [comment]
Closing as wontfix because we're blocked on upstream.
